spec/integration/fieldsets_parser_spec.rb

```diff
@@ -12,7 +12,7 @@
           options do
             fieldsets do
               allowed do
-                posts(schema)
+                resource :posts, schema
```

Comment:
FYI: This was an intentional change in the DSL. Docile does not support those dynamic keys.